catchup/service.go

```diff
@@ -172,7 +173,7 @@ func (s *Service) fetchAndWrite(fetcher rpcs.Fetcher, r basics.Round, prevFetchC
 
 		// Stop retrying after a while.
 		if i > catchupRetryLimit {
-			s.log.Errorf("fetchAndWrite(%v): failed to fetch block many times", )
+			s.log.Errorf("fetchAndWrite(%v): failed to fetch block many times")
```

Comment:
The formatting here is wrong; what's about
"fetchAndWrite: block retrieval exceeded retry limit"